tests/Api/ApiTest.php

```diff
@@ -781,6 +781,11 @@ public function provideServiceClasses()
                 Services\EmailCredentialService::class,
                 Entities\EmailCredential::class,
             ],
+            [
+                'apiKeys',
+                Services\ApiKeyService::class,
+                Entities\ApiKey::class,
+            ],
```

Comment:
Please add Entity test (`provideEntityClasses` method). And add fields to `getFakeValue` method